.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: dart
-dist: precise
+
 dart:
   - "1.24.3"
-env:
-  - DARTIUM_EXPIRATION_TIME="1577836800"
-with_content_shell: true
-before_install:
-  - export DISPLAY=:99.0
-  - ./tool/setup_dartium.sh
-  - export PATH=$PATH":$PWD"
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3 # give xvfb some time to start
+  - stable
+
+# Workaround for issue with sandboxed Chrome in containerized Travis builds.
```

Comment:
Is this needed? didn't need it for `over_react`. Chrome is installed by default in the dart image: https://docs.travis-ci.com/user/languages/dart/#available-browsers